plugins/delpoyment_stats/app/models/deployment_stats.rb

```
@@ -0,0 +1,63 @@
+    init if @@db.nil?
+
+    all_stats = @@db.execute("SELECT * FROM stats").map do |row|
+      process_stat(row)
+    end
+
+    all_stats.map { |stat| OpenStruct.new(stat) }
+  end
+
+  private
+
+  def self.process_stat(row)
```

rubocop (1.68.0)

[warning] `private` (on line 39) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.

(Lint/IneffectiveAccessModifier)